security/passwords.rst

```diff
@@ -697,6 +697,32 @@ you must register a service for it in order to use it as a named hasher:
 This creates a hasher named ``app_hasher`` from a service with the ID
 ``App\Security\Hasher\MyCustomPasswordHasher``.
 
+Hashing a Stand-Alone String
+----------------------------
+
+The password hasher can be used to hash strings independently
+of users. By using the
+:class:`Symfony\\Component\\PasswordHasher\\Hasher\\PasswordHasherFactory`,
+you can retrieve declare multiple hashers, retrieve any of them with
+its name and create hashes. You can then verify that a string matches the given
+hash::
+
+    use Symfony\Component\PasswordHasher\Hasher\PasswordHasherFactory;
+
+    // configure different password hashers via the factory
+    $factory = new PasswordHasherFactory([
+        'common' => ['algorithm' => 'bcrypt'],
+        'sodium' => ['algorithm' => 'sodium'],
+    ]);
+
+    // retrieve the hasher using bcrypt
+    $passwordHasher = $factory->getPasswordHasher('common');
+    $hash = $passwordHasher->hash('plain');
+
+    // verify that a given plain password matches the hash calculated above
```

Comment:
```suggestion
    // verify that a given plain string matches the hash calculated above
```

as it seems you want to document the hashing process of a bare string, not password string

Reply:
Fixed as well, thanks!